.kitchen.yml

```diff
@@ -0,0 +1,142 @@
+---
+driver:
+  name: docker
+  hostname: salt.ci.local
+  use_sudo: false
+
+provisioner:
+  name: salt_solo
+  salt_install: bootstrap
+  salt_bootstrap_url: https://bootstrap.saltstack.com
+  salt_version: latest
+  require_chef: false
+  log_level: error
+  formula: salt
+  grains:
+    noservices: True
+  state_top:
+    base:
+      "*":
+        - salt
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - salt
+          - common
+    common.sls:
+      salt:
+        minion:
+          enabled: true
+          master:
+            host: localhost
+      linux:
+        system:
+          enabled: true
+          name: salt
+          domain: ci.local
+  dependencies:
+    - name: git
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-git.git
+    - name: reclass
+      repo: git
+      source: https://github.com/salt-formulas/salt-formula-reclass.git
+
+verifier:
+  name: inspec
+  sudo: true
+
+platforms:
```

Comment:
We should add more platforms, at least:
- Debian Jessie
- Debian Stretch would be also cool as it will be stable soon
- CentOS 7